pandas/io/sas/sas7bdat.py

```diff
@@ -29,6 +29,31 @@
 from pandas.io.sas.sasreader import ReaderBase
 
 
+def _convert_datetimes(sas_datetimes: pd.Series, unit: str):
+    """
+    Convert to Timestamp if possible, otherwise to datetime.datetime.
+    SAS float64 lacks precision for more than ms resolution so the fit
+    to datetime.datetime is ok.
+
+    Parameters
+    ----------
+    sas_datetimes : Series of 64bit floats representing dates or datetimes
+        in SAS
+    unit : "d" if the floats represent dates, "t" for datetimes
```

Comment:
Description does not match the use below.  unit is either s or d.  Typo?

Also, these are not correctly formatted.  Should be more liek

```
sas_datetime : {Series, Sequence[float]}
   Dates or datetimes in SAS
....
```

Of course this is internal, but might as well be consistent when possible.

Reply:
Good spot - I've altered the comment